pkg/proxy/healthcheck/proxier_health.go

```diff
@@ -156,7 +170,14 @@ type healthzHandler struct {
 }
 
 func (h healthzHandler) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
+	var nodeHealthy bool
```

Comment:
I think we should not talk about node being "healthly" here but "eligible" or "viable" or something?

Reply:
Yeah, I didn't like "healthy" either. I'd like something which goes well with the path: `/livez` - `nodeLive`?

Reply:
It's not about being alive either though - I find "eligible" to be the least awkward so far ?